features/crud.feature

```diff
@@ -358,6 +358,9 @@ Feature: Create-Retrieve-Update-Delete
       "alias": null
     }
     """
+  Scenario: PUT request on custom operations without content
```

Comment:
Indeed it is incorrect. But why a 400 if we allow empty requests for `PUT`, it should be allowed for all `PUT` operations isn't it?